src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1515,7 +1515,9 @@ void Transformations::MainSnippets() {
         [[maybe_unused]] const auto& inferencePrecision = config.inferencePrecision;
         // Note: BrgemmTPP doesn't support transposed KN natively
         // so we should extract transposes for the corresponding matmul nodes
-#if defined(SNIPPETS_LIBXSMM_TPP)
+#if defined(OPENVINO_ARCH_ARM64)
+        return false;
```

Comment:
Could you please add the comment with brief explanation here why we return `false` for arm64? To recall the reason in the future